my-webxr-app/src/App.tsx

```diff
@@ -32,6 +36,19 @@ const endPoint: number = 1;
 const radius: number = 0.002;
 
 export default function App() {
+  // PCA run check
+  const dataset = new Matrix([
+    [1, 2, 3, 4],
+    [5, 5, 6, 7],
+    [1, 4, 2, 3],
+    [5, 3, 2, 1],
+    [8, 1, 2, 2],
+  ]);
+  // eslint-disable-next-line
+  console.log('Covariance PCA:', computeCovariancePCA(dataset, 4));
+  // eslint-disable-next-line
+  console.log('Classic PCA', computeClassicPCA(dataset, 4));
+
```

Comment:
This is good for testing locally, but needs to be removed before merging

Reply:
Good point, fixed